src/store/view-configs/volume-coloring.ts

```diff
@@ -53,8 +53,8 @@ export const defaultVolumeColorConfig = (): VolumeColorConfig => ({
   cvr: {
     enabled: true,
     lightFollowsCamera: true,
-    useVolumetricScatteringBlending: false,
-    volumetricScatteringBlending: 0,
+    useVolumetricScatteringBlending: true,
```

Comment:
LGTM! Do we want to have scattering enabled by default? My only concern is that the initial render will take longer as a result. @aylward

Reply:
IMHO, scattered rendering is the first thing that a user should see. To improve performance, we should disable interactive shading. As for first render, do you know the time difference between cinematic and shaded volume first frames? If that is of  bigger concern, we could look into improving first frame time separately